curiosrc/web/static/sector/index.html

```diff
@@ -0,0 +1,117 @@
+
+  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.min.css" />
+  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.min.js"></script>
+
+  <link rel="stylesheet" href="https://cdn.datatables.net/select/2.0.0/css/select.dataTables.min.css" />
+  <script src="https://cdn.datatables.net/select/2.0.0/js/dataTables.select.min.js"></script>
+</head>
+
+<body style="visibility:hidden" data-bs-theme="dark">
+  <curio-ux>
+    <table id="sectorTable">
+      <tr>
+        <td>Loading...</td>
+      </tr>
+    </table>
+  </curio-ux>
+  <script>
+
+    let dt = new DataTable('#sectorTable', {
+      ajax: '/api/sector/all',
+      columns: [
+        {title: "", data: null},
+        {title: "Miner", data:'MinerID'},
+        {title: "Sector", data:'SectorNum'},
+        {title: "Expiry", data:'ExpiresAt'},
+        {title: "🔗", data:'IsOnChain'},
+        {title: "Has Sealed", data:'HasSealed'},
+        {title: "⨎/Day", data:'ExpectedDayReward'},
+        {title: "Fil+", data:'IsFilPlus'},
+        {title: "Has Unsealed", data:'HasUnsealed'},
+        {title: "Seal Proof", data:'SealProof'},
+        {title: "Has Snap", data:'HasSnap'},
+        {title: "Size", data:"SealInfo"}
+      ],
+      layout: {
+        topStart: 'buttons',
+        bottomStart: 'info',
+      },
+      buttons: [
+        {
+            extend: 'copy',
+            text: '📋'
+        },
+        'csv',
+        {
+            extend: 'selected',
+            text: 'Terminate & Delete',
+            action: function (e, dt, button, config) {
+              var res = dt.rows({ selected: true }).data().map(function (row) {
+                return {MinerID: row.MinerID, Sector: row.SectorNum};
+              });
+
+              if (confirm("Terminate & Delete: "+res.join(", "))) {
+                axios.post('/api/sector/terminate', res)
+                  .then(function (response) {
+                    console.log(response);
+                    document.cookie = "sector_refresh=true; path=/";
+                    location.reload();
+                  })
+                  .catch(function (error) {
+                    console.log(error);
+                  });
+              }
+            }
+        },
+        {
+            text: 'Refresh',
+            action: function (e, dt, button, config) {
+              document.cookie = "sector_refresh=true; path=/";
+              location.reload();
+            }
+        }
+      ],
+      responsive: true,
+      columnDefs: [
+          {
+              orderable: false,
+              render: DataTable.render.select(),
+              targets: 0
+          }
+      ],
+      order: [[1, 'asc']],
+      select: {
+        style: 'multi',
+        selector: 'td:first-child',
+        items: 'row',
+        rows: '%d rows selected',
+        headerCheckbox: true,
+      },
+      scrollY:        200,
```

Comment:
We should probably increase this to 1000. Current scroller is too small.

Reply:
hows  window.innerHeight - 150